src/main/java/com/kodilla/ecommercee/repository/UserRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.kodilla.ecommercee.repository;
+
+import com.kodilla.ecommercee.domain.User;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+@Transactional
+@Repository
+public interface UserRepository extends CrudRepository<User, Long> {
+    List<User> findAll();
```

Comment:
Right now I don't remember why (I will have to check it) but we always used annotation @Override if we implement methods that already are implemented in CrudRepository.

Reply:
Yes, that was written in kodilla course but I don't use it, it works well without it. If you want I can add this.